SiennaOpenAPIModels.jl/Project.toml

```diff
@@ -17,6 +17,7 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 [compat]
 DBInterface = "2.6.1"
 JSON = "0.21.4"
+JuliaFormatter = "1.0.62"
```

Comment:
Please remove this from the Project.toml, since this will make it dependency to anyone calling the project.